cmd/cloud/deployment.go

```diff
@@ -155,6 +155,7 @@ func newDeploymentUpdateCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&label, "name", "n", "", "Update the Deployment's name. If the new name contains a space, specify the entire name within quotes \"\" ")
 	cmd.Flags().StringVarP(&workspaceID, "workspace-id", "w", "", "Workspace the Deployment is located in")
 	cmd.Flags().StringVarP(&description, "description", "d", "", "Description of the Deployment. If the description contains a space, specify the entire description in quotes \"\"")
+	cmd.Flags().StringVarP(&executor, "executor", "e", deployment.CeleryExecutor, "The executor to use for the deployment. Possible values can be CeleryExecutor or KubernetesExecutor.")
```

Comment:
ideally, the update command shouldn't have any default to avoid unintentional updates for the users
```suggestion
	cmd.Flags().StringVarP(&executor, "executor", "e", "", "The executor to use for the deployment. Possible values can be CeleryExecutor or KubernetesExecutor.")
```

Reply:
Good Catch!